### ISSUE TITLE
kraken is no longer brew installable

### ISSUE THREAD

**Issue by davidewatson** [samsung-cnct/homebrew-k2cli#10]
```
rodin:chart-fluent-bit dwat$ brew install k2cli
==> Installing k2cli from samsung-cnct/k2cli
==> Downloading https://github.com/samsung-cnct/k2cli/releases/download/1.1.0/k2cli_1.1.0_darwin_amd64.tar.gz
Already downloaded: /Users/dwat/Library/Caches/Homebrew/k2cli-64.tar.gz
Error: No such file or directory - k2cli
rodin:chart-fluent-bit dwat$ 
```

**Comment by davidewatson** [comment]
This was after uninstalling and untapping the repo.

**Comment by alejandroEsc** [comment]
does install kraken work?

**Comment by davidewatson** [comment]
That is another problem which I forgot to pull out of slack. There is already a kraken command in the science keg.

**Comment by coffeepac** [comment]
would it be best to rename this `kraken-cli`?  I'm not super jazzed about it but it would seem to work.

**Comment by davidewatson** [comment]
We could. The alternative is `brew install samsung-cnct/k2cli/kraken`. I prefer leaving the name and installing it explicitly. The docs need to be updated either way. brew will help you too:

```
rodin:chart-fluent-bit dwat$ brew install kraken
Error: Formulae found in multiple taps: 
 * homebrew/science/kraken
 * samsung-cnct/k2cli/kraken

Please use the fully-qualified name e.g. homebrew/science/kraken to refer the formula.
rodin:chart-fluent-bit dwat$
```

**Comment by davidewatson** [comment]
Fixed by #10 and #11.
